subprojects/persistent-cache/src/main/java/org/gradle/cache/internal/AbstractCacheCleanup.java

```diff
@@ -55,17 +56,18 @@ public boolean accept(File file) {
         });
     }
 
-    protected boolean isReserved(final PersistentCache persistentCache, File file) {
+    static boolean isReserved(final PersistentCache persistentCache, File file) {
```

Comment:
This method probably also should be annotated with `@VisibleForTesting`, right?

Reply:
Fixed.